src/translate.rs

```diff
@@ -1162,38 +1176,64 @@ impl FromGlib<i32> for Option<u64> {
     }
 }
 
-/// Translate from a pointer type without taking ownership, transfer: none.
+/// Translate from a pointer type which is annotated with `transfer: none`.
+/// The resulting value is referenced at least once, by the bindings.
+/// The value must be unreferenced on `drop()`.
+///
+/// This is suitable for floating references, which become full and owned.
```

Comment:
"As soon as g_object_ref_sink() is called on the object, the floating reference is converted to a strong reference, and the calling code assumes ownership of the object. "
"become strong referenced" ?

Reply:
Thanks.